example/pages/product_page.go

```diff
@@ -14,9 +14,16 @@ type ProductPage struct {
 
 func (pp *ProductPage) Containers(r *http.Request) (cs []ct.Container, err error) {
 	cs = []ct.Container{
-		rl.WithReloadEvent("cart_updated", &parts.Header{}),
-		cb.ToContainer(parts.Product),
+		rl.WithTags("cart_updated", &parts.Header{}),
+		&parts.Product{
+			ProductColors: rl.WithTags("cart_updated", &parts.ProductColors{}),
+			ProductImages: &parts.ProductImages{
+				MainImage: rl.WithTags("cart_updated", &parts.ProductMainImage{}),
+			},
+		},
 		cb.ToContainer(parts.Footer),
+		cb.Wrap("script", cb.Attrs{"src": "https://cdnjs.cloudflare.com/ajax/libs/fetch/1.0.0/fetch.min.js"}),
+		cb.ScriptByString(rl.ReloadScript),
 	}
 	return
```

Comment:
The containers tree is setup in a page like this. 
- Header
- Product
  - ProductColors
  - ProductImages
    - MainImage
- Footer
- Script